linux/ibus-keyman/src/engine.c

```diff
@@ -381,6 +382,9 @@ setup_environment(IBusKeymanEngine *keyman)
   environment_opts[2].key   = KM_CORE_KMX_ENV_BASELAYOUTALT;
   environment_opts[2].value = get_base_layout();  // TODO: free when mnemonic layouts are to be supported
 
+  environment_opts[3].scope = KM_CORE_OPT_ENVIRONMENT;
+  environment_opts[3].key   = KM_CORE_KMX_ENV_SIMULATEALTGR;
+  environment_opts[3].value = keyman_get_option_fromdconf(KEYMAN_DCONF_OPTIONS_SIMULATEALTGR) ? u"1" : u"0";
```

Comment:
Ah, is _this_ the part that actually sets the option in Core?  Coming at this again fresh, I think I see now what'd I'd been missing before.

Reply:
yes